buildpack/runtime_components/logs.py

```diff
@@ -36,7 +37,15 @@ def run(self):
             while True:
                 proc = subprocess.Popen(
                     [
-                        "./bin/mendix-logfilter",
+                        str(
+                            os.path.abspath(
+                                os.path.join(
+                                    ".local",
+                                    "mendix-logfilter",
```

Comment:
Is this correct, using `mendix-logfilter` twice?

Reply:
Yes, the binary is inside the `mendix-logfilter` directory. The integration test also calls this binary and it's working.